src/libstd/net/ip.rs

```diff
@@ -822,6 +822,14 @@ impl From<[u8; 16]> for Ipv6Addr {
     }
 }
 
+#[unstable(feature = "ipv6_from_segments", issue = "0")]
```

Comment:
I believe we'll want to mark this stable, since the compiler can't track stability of impls at the moment.